lib/Dancer2/Core/App.pm

```diff
@@ -833,6 +833,41 @@ sub log {
     $logger->$level(@_);
 }
 
+sub send_as {
+    my $self = shift;
+    my ( $type, $data, $options ) = @_;
+    $options ||= {};
+
+    # allow lower cased serializer names
+    my $prefix = 'Dancer2::Serializer::';
+    my $serializer_class;
+    for ( uc $type, $type ) {
```

Comment:
Your assumption is correct. You can always specify the correct 'casing' for the serializer name.

I wanted to allow `json` or `JSON`. Adding `ucfirst` may be useful which would allow `sereal` or `Sereal` if Dancer2::Serializer::Sereal` existed. Not sure its worth adding further heuristics after that.